### DISCUSSION TITLE
I can't get into Github student pack although I have already been approved

### DISCUSSION THREAD

**Discussion by LuongPhuHoa** [community/community#36364] [GitHub Education] [answered]
Hello,

I have registered for GitHub education and got approved, but it does not let me use the education pack. GitHub repeatedly asked me to resubmit the application form and load the page "if I was over 13 or not".

Can anyone help me with this? 

Thank you

![image](https://user-images.githubusercontent.com/36793008/196077635-5284e9d0-6a46-43fe-9c3a-59dae62f7761.png)
![image](https://user-images.githubusercontent.com/36793008/196077778-1e38f81a-bad8-4e63-80e7-92767f1e567d.png)

**Comment by sjkhsl** [comment]
Me too

**Comment by ettaboyle** [comment, marked as answer]
Hi @LuongPhuHoa Thanks for reaching out! You can find all the details you need about the GitHub Student Developer Pack in this post: 
[Student Developer Pack Application & FAQs ➡️ Start Here](https://github.com/orgs/community/discussions/17814)

Please review all the information in that post and follow the steps outlined. Thank you!
